orchestrator/services/workspace_manager.py

```
@@ -0,0 +1,331 @@
+            
+            # Determine destination filename
+            dest_filename = filename or source_path.name
+            dest_path = self.results_dir / dest_filename
+            
+            # Copy file to results directory
+            shutil.copy2(source_path, dest_path)
+            
+            logger.info(f"💾 Saved result: {dest_filename} ({source_path.stat().st_size} bytes)")
+            return str(dest_path)
+            
+        except Exception as e:
```

Ruff (0.14.4)

Do not catch blind exception: `Exception`

(BLE001)